src/renderer/components/playlist-info/playlist-info.vue

```diff
@@ -68,11 +98,71 @@
         </h3>
       </div>
 
-      <ft-share-button
-        v-if="!hideSharingActions"
-        :id="id"
-        :dropdown-position-y="description ? 'top' : 'bottom'"
-        share-target-type="Playlist"
+      <div class="playlistOptions">
+        <ft-icon-button
+          v-if="editMode"
+          :title="$t('User Playlists.Save Changes')"
+          :icon="['fas', 'save']"
+          theme="secondary"
+          @click="savePlaylistInfo"
+        />
+        <ft-icon-button
+          v-if="editMode"
+          :title="$t('User Playlists.Cancel')"
+          :icon="['fas', 'times']"
+          theme="secondary"
+          @click="exitEditMode"
+        />
+
+        <ft-icon-button
+          v-if="!editMode && infoSource === 'user'"
+          :title="$t('User Playlists.Edit Playlist Info')"
+          :icon="['fas', 'edit']"
+          theme="secondary"
+          @click="enterEditMode"
+        />
+        <ft-icon-button
+          v-if="videoCount > 0 && showPlaylists && !editMode"
+          :title="$t('User Playlists.Copy Playlist')"
+          :icon="['fas', 'copy']"
+          theme="secondary"
+          @click="toggleCopyVideosPrompt"
+        />
+        <ft-icon-button
+          v-if="!editMode && infoSource === 'user' && videoCount > 0"
+          :title="$t('User Playlists.Remove Watched Videos')"
+          :icon="['fas', 'eye-slash']"
+          theme="secondary"
+          @click="showRemoveVideosOnWatchPrompt = true"
+        />
+        <ft-icon-button
+          v-if="deletePlaylistButtonVisible"
+          :title="$t('User Playlists.Delete Playlist')"
+          :icon="['fas', 'trash']"
+          theme="secondary"
+          @click="showDeletePlaylistPrompt = true"
+        />
+        <ft-share-button
+          v-if="sharePlaylistButtonVisible"
+          :id="id"
+          :dropdown-position-y="description ? 'top' : 'bottom'"
+          share-target-type="Playlist"
+        />
+      </div>
+
+      <ft-prompt
+        v-if="showDeletePlaylistPrompt"
+        :label="$t('User Playlists.Are you sure you want to delete this playlist? This cannot be undone.')"
+        :option-names="deletePlaylistPromptNames"
+        :option-values="deletePlaylistPromptValues"
+        @click="handleDeletePlaylistPromptAnswer"
+      />
+      <ft-prompt
+        v-if="showRemoveVideosOnWatchPrompt"
+        :label="$t('User Playlists.Are you sure you want to remove all watched videos from this playlist? This cannot be undone.')"
```

Comment:
the . at the end is causing issues when called like this. You can either remove the . from the key or do the following change
```suggestion
        :label="$t('User Playlists[\'Are you sure you want to remove all watched videos from this playlist? This cannot be undone.\']')"
```